frame/transaction-payment/src/lib.rs

```diff
@@ -178,7 +172,8 @@ impl<T: Trait> Module<T> {
 			let targeted_fee_adjustment = NextFeeMultiplier::get();
 			let adjusted_fee = targeted_fee_adjustment.saturated_multiply_accumulate(adjustable_fee.saturated_into());
 
-			let base_fee = T::TransactionBaseFee::get();
+			let base_weight = T::ExtrinsicBaseWeight::get();
+			let base_fee = Self::weight_to_fee(base_weight);
```

Comment:
Same as above. I would rather see `base_fee` calculated in one expression. This intermediate binding is redundant and is just one variable more that I have to keep in my mind.